client/tasks/fills/PaymentGatewaySuggestions/components/WCPay/Suggestion.js

```diff
@@ -82,6 +83,11 @@ export const Suggestion = ( { paymentGateway, onSetupCallback = null } ) => {
 							'woocommerce-admin'
 						) }
 						onSetupCallback={ onSetupCallback }
+						onSetUp={ () =>
```

Comment:
Should we pass down something like `defaultOnSetUp` here so that SlotFills can just call that avoiding any breakage in potential future updates?

We have some precedence here with other gateway props.

Reply:
Unless we just plop the track logic into the `Action` component here: https://github.com/woocommerce/woocommerce-admin/blob/main/client/tasks/fills/PaymentGatewaySuggestions/components/Action.js#L35

And use the `onSetup` purely as a notifier callback. Then we could remove it's use here as well:
```
https://github.com/woocommerce/woocommerce-admin/blob/main/client/tasks/fills/PaymentGatewaySuggestions/components/List/Item.js#L89-L93
```

Reply:
Great solution, @louwie17!  That simplifies things a bit and looks great to me.